pocketimdb/movies/models.py

```diff
@@ -23,3 +23,12 @@ class MovieLike(models.Model):
 
     class Meta:
         unique_together = ('movie', 'user',)
+
+class MovieComment(models.Model):
+    movie = models.ForeignKey(Movie, on_delete=models.CASCADE, related_name='movie_comments')
+    user = models.ForeignKey(User, on_delete=models.CASCADE, related_name='user_comments')
+    comment = models.CharField(max_length=500)
+    timestamp = models.DateTimeField(auto_now_add=True)
```

Comment:
ovo je predpotsavljam kada je kreiran komentar, tako da mozs nazvati polje `created_at` i isto tako mozes dodatni `update_at` za to kada je azuzirano ukoliko ce psotojati tako nesto 
```
created_at = models.DateTimeField(auto_now_add=True, editable=False, null=False)
updated_at = models.DateTimeField(auto_now=True, editable=True, null=False)
```

Reply:
Da, to se odnosi na to kada je kreiran komentar. Semanticki mi dosta vise odgovara da se to zove timestamp.
Ali mogu izmeniti ukoliko je potrebno.
`updated_at` polje mi ne treba u ovom slucaju, nikada se komentar ne menja jer ne postoji takva funkcionalnost.

Reply:
jasno, samo sto ce sutra biti i update, a ti imas timestamp koji moze znaciti vise stvari pa i update_at, tako da hajde da promjenimo u created_at

Reply:
Promenio sam, shvatio sam u medjuvremenu da ipak ima vise smisla da bude created_at. A verujem da je i praksa da se tako naziva to polje.